magicgui/widgets/_concrete.py

```diff
@@ -749,28 +750,43 @@ def annotation(self, value):
         self._annotation = value
         self._args_type = arg
 
+    def __iter__(self) -> Iterator[ValueWidget]:
```

Comment:
Container is now Generic as of #475 ... so these aren't necessary anymore if we declare `class ListEdit(Container[ValueWidget])` ... but I'll do that in a followup PR